backend/app/main.py

```
@@ -46,33 +123,110 @@ def create_app() -> FastAPI:
-    return templates.TemplateResponse(
-        "index.html", 
-        {"request": request, "app_name": settings.PROJECT_NAME}
-    )
+async def dashboard(request: Request):
+    return templates.TemplateResponse("index.html", {"request": request})
 
 
-# Frontend routes that should return the SPA template
+# Individual page routes
 @app.get("/dashboard", response_class=HTMLResponse)
+async def dashboard(request: Request):
```

Ruff (0.8.2)

Redefinition of unused `dashboard` from line 126

(F811)